addons/storyshots/storyshots-puppeteer/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../../tsconfig.json",
   "compilerOptions": {
     "rootDir": "./src",
+    "skipLibCheck": true,
```

Comment:
Storyshots is old, and contains code for multiple frameworks, it gets 'confused' about JSX and frameworks fight over it. Let's skip this for now.